feature_tests/environment.py

```diff
@@ -12,12 +12,17 @@ def before_all(context: Context):
 
 def before_scenario(context: Context, scenario: Scenario):
     context.test_config = config_setup(context=context, scenario_name=scenario.name)
+    if (
+        "integration-only" in scenario.tags
+        and context.test_config.mode is TestMode.LOCAL_TEST
+    ):
+        scenario.mark_skipped()
 
 
 def after_scenario(context: Context, scenario: Scenario):
     test_config: TestConfig = context.test_config
-    # for repository in test_config.repositories.values():
-    #     repository.delete_all()
+    for repository in test_config.repositories.values():
```

Comment:
Now an extra scary thing to make sure we dont run in Prod haha

Reply:
ah it happens anyway, just making sure it happens after the tests as well as before